features/Pim/Behat/Context/Domain/SecondaryActionsContext.php

```diff
@@ -16,26 +17,92 @@ class SecondaryActionsContext extends PimContext
      */
     public function iPressTheSecondaryAction($actionName)
     {
-        $module = $this->spin(function () {
+        $module = $this->getModule();
+        $this->openModule($module);
+        $this->spin(function () use ($module, $actionName) {
+            return $this->getAction($module, $actionName);
+        }, sprintf('Could not find the secondary action "%s"', $actionName))->click();
+    }
+
+    /**
+     * @param string $not
+     * @param string $actionName
+     *
+     * @Then /^I should (not )?see the secondary action "(?P<actionName>.*)"$/
+     */
+    public function iShouldSeeTheSecondaryAction($not, $actionName)
+    {
+        $module = $this->getModule();
+        $this->openModule($module);
+        $this->spin(function () use ($not, $module, $actionName) {
+            if ('' === $not) {
+                return null !== $this->getAction($module, $actionName);
+            } else {
+                return null === $this->getAction($module, $actionName);
+            }
+        }, sprintf('Secondary action "%s" was %sfound', $actionName, $not));
```

Comment:
In fact, if I will be completely right, I should write
`$not !== 'not ' ? '' : 'not '`
which is stupid.